### ISSUE TITLE
Low validation accuracies when training on all data

### ISSUE THREAD

**Issue by baselmousi** [faezetta/VMMRdb#5]
Hello, 
I hope that you are doing well.
I am trying to use the vmmrdb dataset to classify cars into their corresponding make, model, and year. I'm using the whole dataset (all 9170 classes) with a validation split of 0.2. However I am achieving very low validation accuracies (5-7%). I am using resnet50 with the pre-trained imagenet weights. I'm removing the layer and I'm adding a new dense layer with number of neurons = number of classes. I'm only training the last layer. I'm training for 200 epochs with a learning rate of 0.001. 
I guess that I'm facing the multiplicity and ambiguity problems that are mentioned in the introduction of the paper so I was wondering if it's possible to have have access to the code that you used for training.This will help me in knowing the problem that i'm actually facing better.
Thanks in advance
